src/v/kafka/protocol/errors.h

```diff
@@ -216,6 +216,8 @@ enum class error_code : int16_t {
     // Consumer group The consumer group has reached its max size. already has
     // the configured maximum number of members.
     group_max_size_reached = 81,
+    // No partition reassignment is in progress
+    no_reassignment_in_progress = 85,
```

Comment:
I see you've introduced this new error code but its not used anywhere in this commit

Reply:
hmm, I must've removed the code that used this. I'll change.